src/Bicep.Core.IntegrationTests/ExtensibilityTests.cs

```diff
@@ -16,16 +13,11 @@
 namespace Bicep.Core.IntegrationTests
 {
     [TestClass]
-    public class ExtensibilityTests
+    public class ExtensibilityTests : TestBase
     {
-        [NotNull]
-        public TestContext? TestContext { get; set; }
-
         private ServiceBuilder Services => new ServiceBuilder()
```

Comment:
nit: Not introduced by this PR, but we can make this static to avoid creating a Services instance for each test case execution.

Reply:
I've actually been deliberately avoiding that because there should be isolation between each test. Some of the dependencies being set up are mutable (e.g. mock file system, registry clients).